src/domain/IndexPage/IndexPage.tsx

```diff
@@ -45,6 +46,9 @@ const IndexPage: React.FC = () => {
           {t('index.see_all_button')}
         </Link>
       </HeroBanner>
+      <section className={styles.section}>
```

Comment:
This section should be below the first section and have an appropriate `h2`.

Reply:
done